src/main/scala/org/allenai/plugins/NodeJsPlugin.scala

```diff
@@ -113,26 +92,56 @@ object NodeJsPlugin extends AutoPlugin {
     // we don't care if the command fails here because we are just
     // passing through to the local `npm`
     try {
-      val (newState, env) = extracted.runTask(environment in Npm, state)
+      val env = extracted.getOpt(environment in Npm).get
       exec(args.mkString(" "), extracted.getOpt(nodeProjectDir in Npm).get, env)
-      newState
+      state
     } catch {
       case NpmMissingException => state.fail
       case t: Throwable => state
     }
   }
 
+  /** @returns the node build environment variables, with the given NODE_ENV and NODE_BUILD_DIR set
+    */
+  def getEnvironment(nodeEnv: String, buildDir: File): Map[String, String] = {
+    Map(
+      "NODE_ENV" -> nodeEnv,
+      "NODE_API_HOST" -> "/api",
```

Comment:
I actually have no idea what this is, I just kept what was already there. :)

I feel like there might be a few places the node plugin could be made better. OK with leaving as-is, then?